ports/templatetags/url_generate.py

```diff
@@ -4,9 +4,11 @@
 
 @register.simple_tag
 def build_url(builder_name, build_id):
-    return "https://build.macports.org/builders/ports-" + builder_name + "-builder/builds/" + str(build_id)
+    url_prefix = "https://build.macports.org/builders/ports-"
+    return url_prefix + builder_name + "-builder/builds/" + str(build_id)
 
 
 @register.simple_tag
 def watcher_url(builder_name, watcher_id):
-    return "https://build.macports.org/builders/ports-" + builder_name + "-watcher/builds/" + str(watcher_id)
+    url_prefix = "https://build.macports.org/builders/ports-"
```

Comment:
Actually, the entire point of having the base URL in a variable was to avoid changing the URL in the future at multiple places.

Reply:
Sorry, my bad. Improved this.